.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["next/core-web-vitals", "next/typescript"],
   "rules": {
-    "@typescript-eslint/no-explicit-any": "off"
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Disabling the `@typescript-eslint/no-unused-vars` rule is not recommended. This rule helps to identify declared variables, functions, or function arguments that are never used in your code, which can lead to cleaner and more maintainable code. If there are specific instances where you need to declare a variable without using it (for example, when destructuring an object), consider disabling the rule for that specific line instead of globally.

```diff
{
   "extends": ["next/core-web-vitals", "next/typescript"],
   "rules": {
       "@typescript-eslint/no-explicit-any": "off",
-      "@typescript-eslint/no-unused-vars": "off"
   }
}
```
Please ensure to add a newline at the end of the file as it's a common convention to end a file with a newline. This is due to some tools not being able to handle the last line properly if it doesn't end with a newline.
```diff
{
   "extends": ["next/core-web-vitals", "next/typescript"],
   "rules": {
       "@typescript-eslint/no-explicit-any": "off",
-      "@typescript-eslint/no-unused-vars": "off"
   }
}
+ 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->